addons/knobs/src/registerKnobs.ts

```diff
@@ -37,10 +37,12 @@ function knobChanged(change: KnobStoreKnob) {
   const knobOptions = knobStore.get(name);
   knobOptions.value = value;
 
-  if (!manager.options.disableDebounce) {
-    debouncedResetAndForceUpdate();
-  } else {
-    resetAndForceUpdate();
+  if (!manager.options.disableForceUpdate && !knobOptions.disableForceUpdate) {
+    if (!manager.options.disableDebounce && !knobOptions.disableDebounce) {
```

Comment:
This looks like a breaking change - disableDebounce will only be reachable if disableForceUpdate is set.

Reply:
Actually, resetAndForceUpdate() is reachable when both disableForceUpdate options aren't truthy (aren't set). If we wanna disable force-updating, then we don't need to debounce and run it.

Reply:
Actually, resetAndForceUpdate() is reachable when both disableForceUpdate options aren't truthy (aren't set). If we wanna disable force-updating, then we don't need to debounce and run it.